syz-verifier/stats.go

```diff
@@ -16,12 +16,13 @@ import (
 // of the verification process.
 type Stats struct {
 	// Calls stores statistics for all supported system calls.
-	Calls            map[string]*CallStats
-	TotalMismatches  int64
-	TotalProgs       int64
-	FlakyProgs       int64
-	MismatchingProgs int64
-	StartTime        time.Time
+	Calls               map[string]*CallStats
+	TotalCallMismatches int64
```

Comment:
All stats changes I think can go into a separate commit. That's another -2 files for the PR.

Reply:
I added ExecErrorProgs here.
Question: is it ok to add some TODOs in the PR that are irrelevant to the current PR scope?